crates/solvers/src/domain/solver/balancer.rs

```diff
@@ -0,0 +1,56 @@
+//! A simple DEX solver that matches orders directly with swaps from the
+//! Balancer SOR API
+
+use {
+    crate::{
+        domain::{
+            auction,
+            dex::{self, slippage},
+            solution,
+        },
+        infra,
+    },
+    futures::{future::BoxFuture, FutureExt},
+};
+
+pub struct Balancer {
+    /// The Balancer SOR API client.
+    pub sor: infra::dex::balancer::Sor,
```

Comment:
Not sure if `sor` is known widely enough to not cause confusion here. It doesn't get used a lot so `single_order_router` should be fine IMO.

Reply:
I moved the doc-comment in parent.